.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - *attach_workspace
       - run:
           name: Install Serverless CLI and dependencies
-          command: sudo npm i -g serverless@"<2.0.0"
+          command: sudo npm i -g serverless@3
```

Comment:
Did you test that the pipelien works with the change? Serverless v3 has caused issues in the past: https://github.com/mattermost/mattermost-marketplace/issues/118

Reply:
@hanzei Thanks. I'm not sure how to run the pipeline without committing and trying. Please let me know if you know of another way.

Reply:
If you go to a the pipeline run on `master` https://app.circleci.com/pipelines/github/mattermost/mattermost-marketplace/979/workflows/2fdc5893-9ad2-4a0f-bd87-4590302ba0b9/jobs/1273, you can run a job with SSH and then SSH onto the machine
![Screenshot from 2023-02-02 10-07-19](https://user-images.githubusercontent.com/16541325/216280059-b1561a04-2d26-43a7-8bc3-e72bba2b370e.png)

That allows you to test your changes.

Reply:
Please let me know if you nee another guidance.

Reply:
Yep, I know about rerunning workflows. The workflow that is failing is the `deploy`, which this PR doesn't have. So rerunning wouldn't test it. I guess I'll add deploy to this branch to test it, and then remove.

Reply:
@hanzei Ok. SSHing won't help if the deploy workspace isn't there, so I added the deploy branch to this PR, and tested twice -- serverless installs without a problem. However, I can't verify the rest of the deploy script works in this branch. I suggest we merge to master and see if it completes there.